.golangci.yaml

```diff
@@ -12,6 +12,7 @@ linters:
   disable:
   - maligned
   - scopelint
+  - gci
```

Comment:
Not a fan of `gci`. Do we need it?

Reply:
I disabled it similar to what tim did in your PR.